docs/en/typography.md

```diff
@@ -0,0 +1,92 @@
+# Typography
+The typograhy consists of the following elements: h1, h2, h3, h4, h5, h6, p, ul and ol. We use `em` for font sizing, where `1em` is the default browser font size, which should equal 16px, 12pt or 100%.
+
+Each element's font size can be set per device breakpoint.
+
+Font Family's are passed in to the framewerk by name reference and should be loaded outside of the framewerk.
+
+## Headings: h1-h6
+
+The default heading properties apply `HeadingColor` and `HeadingMargin` to all headings (h1-h6)
+```
+// Default form checkbox Properties
+$default-component-typography-properties:(
+    "HeadingColor": getThemeProperty(textColorDark),
+    "HeadingMargin": 0 0 getThemeProperty(layoutSpace)/2 0
+);
+```
+
+All headings (h1-h6) can be individually customized, example for h1:
+```
+$default-element-h1-properties:(
```

Comment:
I find it just a little odd that these first two sections are missing the comment just above this. We should add them here or remove them all.
See https://github.com/werkbot/framewerk/blob/updates-032122/docs/en/typography.md